carma_cooperative_perception/src/msg_conversion.cpp

```diff
@@ -244,7 +244,21 @@ auto to_detection_list_msg(
 
     detection.header.stamp = to_time_msg(detection_time);
 
-    detection.id = std::to_string(common_data.detected_id.object_id);
+    static constexpr auto to_string = [](const std::vector<std::uint8_t> bsm_id) {
```

Comment:
it sounds like this is `uuid` instead of bsm_id, so let's rename it?

Reply:
I took a look at the J2735 document and saw the type for that message was TemporaryID. I've updated the terminology to reflect that.